forum/qa-plugin/user-changeable-theme/themes/dark/src/elements/aside/_helion.scss

```diff
@@ -0,0 +1,22 @@
+// welp - probably MZ's HTML
+[style='padding:10px; background-color:#fff; font-size: 17px;margin-bottom:5px; text-align:center;']{
```

Comment:
W zasadzie to już jest naprawione, bo obecnie widgetu z okazją od Heliona nie ma - wróci jak napiszemy obsługę z JSONa i wtedy pewnie kod będzie inny/porządniejszy.